src/components/Characters/Characters.js

```diff
@@ -1,6 +1,6 @@
 import CharacterCard from "../CharacterCard/CharacterCard"
 import "./Characters.css"
-import { Link } from "react-router-dom"
+import { NavLink, Link } from "react-router-dom"
```

Comment:
NavLink is a special type of Link that can be styled automatically when the route it links to is active. This is particularly useful for indicating which page or section the user is currently viewing.